docs/Hooks.md

```diff
@@ -141,6 +141,48 @@ fastify.addHook('onSend', (request, reply, payload, next) => {
 
 Note: If you change the payload, you may only change it to a `string`, a `Buffer`, a `stream`, or `null`.
 
+### Respond to a request from an hook
+If need you can respond to a request before you reach the route handler, an example could be an authentication hook. If you are using `onRequest` or a middleware just use `res.end`, if you are using the `preHandler` hook use `reply.send`. Remember to always call `next` if you are using the standard hook api, if you are working with *async* hooks it will be done automatically by Fastify.
```

Comment:
This is not true. If you 'forget' to call `next`, the only harm you will cause is to _potentially_ increase the memory consumption, and you will prevent one of our optimizations to kick in.